products/jbrowse-web/src/sessionWarningModal.tsx

```diff
@@ -54,31 +54,27 @@ export default function SessionWarningModal({
         <WarningIcon fontSize="large" />
         <DialogContent>
           <DialogContentText>
-            External sessions can contain code that runs.
+            This link contains an external session, which may contain dangerous
+            code.
           </DialogContentText>
           <DialogContentText>
-            Please ensure and confirm you trust the source of this session.
+            Please ensure you trust the source of this session.
           </DialogContentText>
         </DialogContent>
         <div className={classes.buttons}>
           <Button
             color="primary"
             variant="contained"
             style={{ marginRight: 5 }}
-            onClick={() => {
-              sessionTriaged.origin === 'share'
-                ? loader.setSessionSnapshot({
-                    ...session,
-                    id: shortid(),
-                  })
-                : loader.setConfigSnapshot({
-                    ...session,
-                    id: shortid(),
-                  })
+            onClick={async () => {
+              loader.setSessionSnapshot({
+                ...session,
+                id: shortid(),
+              })
               handleClose()
```

Comment:
Likewise, should probably pass`onConfirm` and `onCancel` callbacks to this component